internal/cmd/verify.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/sqlc-dev/sqlc/internal/bundler"
+	"github.com/sqlc-dev/sqlc/internal/quickdb"
+	quickdbv1 "github.com/sqlc-dev/sqlc/internal/quickdb/v1"
+)
+
+var verifyCmd = &cobra.Command{
+	Use:   "verify",
+	Short: "Verify schema, queries, and configuration against main",
```

Comment:
I don't think "against main" means anything here. I don't have a better idea at the moment, but I think just saying `"Verify schema, queries, and configuration for this project"` would be fine for now.